test/it/uk/gov/hmrc/exports/migrations/changelogs/notification/MakeParsedDetailsOptionalSpec.scala

```diff
@@ -13,7 +14,7 @@ import stubs.TestMongoDB.mongoConfiguration
 import uk.gov.hmrc.exports.base.UnitSpec
 import uk.gov.hmrc.exports.migrations.changelogs.notification.MakeParsedDetailsOptionalSpec._
 
-class MakeParsedDetailsOptionalSpec extends UnitSpec with GuiceOneServerPerSuite {
+class MakeParsedDetailsOptionalSpec extends UnitSpec with GuiceOneServerPerSuite with IntegrationPatience {
```

Comment:
Added the trait `IntegrationPatience` trait to a few integration tests to avoid potential random timeout exceptions due to the small default (150 millisecs)

Reply:
You've changed `IntegrationSpec` earlier and it mixes-in `IntegrationPatience`. I'd suggest simply extending this spec instead. 
But because WirerMock is not required in all Integration Tests, I'd also remove `MockGenericDownstreamService` from there. This would be only to speed tests up as I'd expect WireMock to take some time to launch.